pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -1510,6 +1510,70 @@ PJ_DEF(pj_status_t) pjsua_acc_modify( pjsua_acc_id acc_id,
     return status;
 }
 
+typedef struct pjsua_send_request_data
+{
+    pjsua_acc_id acc_id;
+    void *token;
+} pjsua_send_request_data;
```

Comment:
A minor: by convention, `pj*_` prefixes should only be used for API. As this is internal type, it should not use `pjsua_` prefix (e.g: just `send_request_data`).